client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
I think `error` and `warning` could be separate log levels from `server-warning` and `server-error`. And that it shouldn’t print disconnected more than once regardless of the log level. Ideally it wouldn’t print failed `GET *` either—not sure if there’s a way to prevent this.